assets/other-scripts/recaptcha/index.js

```diff
@@ -34,89 +24,180 @@ function domReady( callback ) {
 	document.addEventListener( 'DOMContentLoaded', callback );
 }
 
+window.newspack_grecaptcha = window.newspack_grecaptcha || {
+	destroy,
+	render,
+	version: newspack_recaptcha_data.version,
+};
+
+const isV2 = 'v2' === newspack_recaptcha_data.version.substring( 0, 2 );
+const isV3 = 'v3' === newspack_recaptcha_data.version;
```

Comment:
Nonblocking nitpick: I guess we can save the v2 calculation by assuming if `isV3` is false, `isV2` must be true? Not sure if `newspack_recaptcha_data.version` can be anything else though.

Reply:
We originally did build in support for the non-invisible (checkbox) flavor of v2, which you might have seen on sites where you have to click a button that says "I am not a robot" to trigger the verification. So that's why all of the logic is written as if there are more than two options. It feels like it might be more futureproof to expect that there could be more than two options, too, because maybe Google will release a v4 at some point?